package.json

```diff
@@ -42,7 +42,7 @@
     "@glimmer/tracking": "^1.0.1",
     "@html-next/vertical-collection": "^1.0.0",
     "amd-name-resolver": "^1.3.1",
-    "aws-sdk": "^2.729.0",
+    "aws-sdk": "^2.731.0",
```

Comment:
Seems unrelated?

Reply:
Yes, this one is unrelated, but just bumped the couple deps that were not up to date.